components/brave_new_tab_ui/components/default/braveToday/useVisible.ts

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react'
```

Comment:
This seems like something worth moving to `brave/components/web-common` as it's very generic. We already have the similar `VisibilityTimer` (which I'm not necessarily suggesting merging this with as `useVisible` is react specific and has a threshold (not that `VisibilityTimer` couldn't get a threshold param!)

Reply:
:+1: done!